assets/scss/_variables_project.scss

```diff
@@ -38,10 +38,17 @@ p {
   text-align: center;
 
   .banner-image {
+    display: block;
     margin-bottom: 2em; 
     margin-left: auto;
     margin-right: auto;
-    max-width: 65%;
+    max-width: 75%;
+  }
+  
+  .banner-image img {
```

Comment:
nit: Could fold this into the `.banner-image` block.

Reply:
Done.